plugins/binanceExchange_ws.go

```diff
@@ -286,6 +350,106 @@ func (beWs *binanceExchangeWs) GetTickerPrice(pairs []model.TradingPair) (map[mo
 	return priceResult, nil
 }
 
+//GetOrderBook impl
+func (beWs *binanceExchangeWs) GetOrderBook(pair *model.TradingPair, maxCount int32) (*model.OrderBook, error) {
+
+	var (
+		maxCountInt = int(maxCount)
+		fetchLimit  = maxCountInt
+	)
+
+	if fetchLimit > 20 {
+		fetchLimit = 20
```

Comment:
can throw an error here saying that this is invalid and max supported value is 20